src/model/HeatPumpAirToWaterCooling.cpp

```diff
@@ -348,9 +348,21 @@ namespace model {
       return result;
     }
 
+    boost::optional<double> HeatPumpAirToWaterCooling_Impl::autosizedRatedCoolingCapacity() const {
+      boost::optional<double> result;
+      if (auto awhp_ = heatPumpAirToWater()) {
+        result = awhp_->autosizedRatedCoolingCapacity();
+      }
+      return result;
+    }
+
     void HeatPumpAirToWaterCooling_Impl::autosize() {
       autosizeRatedAirFlowRate();
       autosizeRatedWaterFlowRate();
+      auto speeds = this->speeds();
+      if (!speeds.empty()) {
+        speeds.back().autosize();  // Only need/can autosize the last (highest) speed
+      }
```

Comment:
in autosize, this is weird, but I autosize the last speed.

The way Model::autosize/applySizingValues works is that it does it for HVACComponent, and the speed data is not one and we do not want to add an explicit case because none of the speeds should be autosized except the last (highest) one